paddle/scripts/paddle_build.sh

```diff
@@ -441,7 +442,9 @@ EOF
         # make install should also be test when unittest
         make install -j 8
         if [ "$1" == "cp27-cp27m" ]; then
+            set -e
             pip install --user ${INSTALL_PREFIX:-/paddle/build}/opt/paddle/share/wheels/*.whl
+            python ${PADDLE_ROOT}/paddle/scripts/installation_validate.py
```

Comment:
Need some validation here?

Reply:
yes, this is used to validate `import paddle.fluid` here for now, and we can add more validate here later